src/config/bootloader.rnc

```diff
@@ -104,7 +104,8 @@ section =
     element remap { "true" | "false" }? &
     element configfile {text}? &
     element image   {text}? &
-    element description {text}?
+    element description {text}? &
+    element nounzip {text}?
```

Comment:
I think we should port this change to master according to https://github.com/openSUSE/perl-bootloader/search?utf8=%E2%9C%93&q=nounzip